charts/azure-managed-cluster/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v2
+name: azure-managed-cluster
+description: A Helm chart to deploy an AKS cluster via the cluster-api-provider-azure ManagedCluster
+maintainers:
+  - name: nellyk
```

Comment:
@dtzar  i would need to add a maintainer here to get this to pass. It would help though if i can get some names to add here